changelog.md

```diff
@@ -1,5 +1,11 @@
 ## ROC'n'ROLL
 
+### Version 0.6.3
```

Comment:
In the future, you don't need to break down the changes of a single version into multiple versions. One new version can contain everything you changed. ;D